website/docs/dbt-cli/install/from-source.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Install from source"
+id: "from-source"
+---
+
+dbt Core is open source software, as are almost all adapter plugins. As such, the codebases are freely available to download and build from source. To do this, you can clone the repositories from GitHub and install the local version using `pip`.
```

Comment:
>as are almost all adapter plugins

Which ones aren't OSS? How would someone even install those?

Reply:
yeah, to be clear - if they're not OSS, we (probably) don't know about them! and the only people who can install them are the people who built them, or their colleagues. but, in theory, it's possible that there exist closed-source adapters (and we know of a handful).